src/components/companies/Companies.js

```diff
@@ -56,61 +57,65 @@ export default class Companies extends Component {
         })
     }
 
-    editItem({id, name, address, phonesNumber, site, description}) {
-        const newItem = {
-            name,
-            address,
-            phonesNumber,
-            site,
-            description
-        }
-        this.setState(({date}) => {
-            const index = date.findIndex(elem => elem.id === id);
-
-            const before = date.slice(0, index);
-            const after = date.slice(index + 1);
-            
-            const newArr = [...before, newItem, ...after];
-            return {
-                date: newArr
+    editItem({ name, address, phonesNumber, site, description, id}) {
+            const newItem = {
+                name,
+                address,
+                phonesNumber,
+                site,
+                description,
+                id: this.maxId++
             }
-        });
+            this.setState(({date}) => {
+                const index = date.findIndex(elem => elem.id === id);
+
+                const before = date.slice(0, index);
+                const after = date.slice(index + 1);
+                
+                const newArr = [...before, newItem, ...after];
+                return {
+                    date: newArr
+                }
+            });
     }
 
     
 
-    searchPost(items, term) {
-        if (term.lenght === 0) {
+    searchPost(items, search) {
+        if (search.lenght === 0) {
             return items
         }
 
         return items.filter( (item) => {
-            return item.name.indexOf(term) > -1
+            return item.name.indexOf(search) > -1
         });
     }
 
-    onUpdateSearch(term) {
-        this.setState({term});
+    onUpdateSearch(search) {
+        this.setState({search});
     }
 
 
     render() {
-        const { date, term } = this.state;
+        const { date, search } = this.state;
 
-        const visiblePosts = this.searchPost(date, term);
+        const visiblePosts = this.searchPost(date, search);
```

Comment:
1. Finish removing 'post' word;
2. Use getter instead of method:
```
    get companiesShownList() {
        const allCompanies = this.state.data;
        const searchString = this.state.search;

        if (searchString === 0) {
            return allCompanies;
        }

        return [...allCompanies].filter(({name}) => {
            return name.indexOf(searchString) > -1
        });
    }
```

Reply:
done

Reply:
Not done. Check it again